Formula/ooniprobe.rb

```diff
@@ -152,36 +189,32 @@ def install
     (buildpath/"ooni/settings.ini").atomic_write <<-EOS.undent
       [directories]
       usr_share = #{share}/ooni
-      var_lib = #{var}/lib/ooni
+      var_lib = #{var}/run/ooni
+      etc = #{etc}/run/ooni
     EOS
 
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
     system "python", *Language::Python.setup_install_args(libexec)
     bin.install Dir["#{libexec}/bin/*"]
-    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"], :PATH => ENV["PATH"])
 
     man1.install Dir["data/*.1"]
-    (share/"ooni").install Dir["data/*"]
+    (share/"ooni").install Dir["data/*.conf.sample"]
+    (share/"ooni/decks-available").install Dir["data/decks/*"]
   end
 
-  def post_install
-    require "open3"
+  def caveats; <<-EOS.undent
+    If you are updating from an older version of ooniprobe be sure to run:
+       brew services restart ooniprobe
 
-    (var/"lib/ooni").mkpath
-    system bin/"ooniresources"
-    Open3.popen3("#{bin}/oonideckgen", "-o",
-                 "#{HOMEBREW_PREFIX}/share/ooni/decks/") do |_, stdout, _|
-      current_deck = stdout.read.split("\n")[-1].split(" ")[-1]
-      ln_s current_deck, "#{HOMEBREW_PREFIX}/share/ooni/decks/current.deck"
-    end
-  end
+    If this is your first time installing ooniprobe run:
+       brew services start ooniprobe
 
-  def caveats; <<-EOS.undent
-    Decks are installed to #{HOMEBREW_PREFIX}/share/ooni/decks/.
+    The you will be able to access the ooniprobe web user interface by going to http://127.0.0.1:8842/
     EOS
```

Comment:
Can I leave the message with information on how to access the web UI, but just drop the ones regarding the services? Is there some better way to give users post installation instructions?